__tests__/app.test.js

```diff
@@ -200,6 +200,39 @@ describe("/api/articles", () => {
             });
          });
    });
+
+   describe("GET Request: Sorting queries", () => {
+      test("GET 200: Should respond with all articles sorted by votes and by default: in descending order", () => {
+         return request(app)
+            .get("/api/articles?sort_by=votes")
+            .expect(200)
+            .then(({ body: { articles } }) => {
+               expect(articles).toBeSortedBy("votes", {
+                  descending: true,
+               });
+            });
+      });
+
+      test("GET 200: Should respond with all articles sorted by votes and in ascending order (case insensitive)", () => {
+         return request(app)
+            .get("/api/articles?sort_by=votes&order=ASC")
+            .expect(200)
+            .then(({ body: { articles } }) => {
+               expect(articles).toBeSortedBy("votes", {
+                  ascending: true,
+               });
+            });
+      });
+
+      test("GET 400: Should respond with err message 'Invalid query' when passed an invalid sort_by or order value", () => {
```

Comment:
I would recommend splitting this test into two whereby you test sort_by and order separately